RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETMember.cs

```diff
@@ -15,7 +15,9 @@ public class CaDETMember
         public ISet<CaDETMember> InvokedMethods { get; internal set; }
         public ISet<CaDETMember> AccessedAccessors { get; internal set; }
         public ISet<CaDETField> AccessedFields { get; internal set; }
+        public CaDETClass ReturnType { get; internal set; }
         public List<string> VariableNames { get; internal set; }
```

Comment:
VariableName and VariableType should be encapsulated into a CaDETVariable. Note that this will cause compilation errors in the BasicNamingStrategy of the SmartTutor so modify that code as well and run tests for both RC and ST.